python/trainer_config.yaml

```diff
@@ -2,14 +2,14 @@ default:
     trainer: ppo
     batch_size: 1024
     beta: 5.0e-3
-    buffer_size: 10000
+    buffer_size: 10240
     epsilon: 0.2
     gamma: 0.99
     hidden_units: 128
     lambd: 0.95
     learning_rate: 3.0e-4
     max_steps: 5.0e4
-    normalize: false
+    normalize: true
```

Comment:
Does this help in all cases ? Should the default really be true ?

Reply:
You are right. Better to set it to false by default.